### PR TITLE
StatusBar

### PR DESCRIPTION
**Pull request by CristianRWolox**
## Summary

- Implement StatusBar component.

#### Galaxy S (or similar)
Captura de Pantalla 2021-06-04 a la(s) 6 51 42 a  m

#### Pixel 2/3 (or similar)
Captura de Pantalla 2021-06-04 a la(s) 6 52 34 a  m

#### Pixel 3XL (or similar)
Captura de Pantalla 2021-06-04 a la(s) 6 47 50 a  m

#### iPhone SE (or similar)
Captura de Pantalla 2021-06-04 a la(s) 6 30 12 a  m

#### iPhone 6/7/8 (or similar)
Captura de Pantalla 2021-06-04 a la(s) 6 54 57 a  m

#### iPhone 11/12 Base/Pro/Pro Max (or similar)
Captura de Pantalla 2021-06-04 a la(s) 6 56 28 a  m

## Trello Card

[StatusBar](https://trello.com/c/aWnXy0O4/58-n-15-statusbar)

### CODE REVIEW

File: wbooks/src/app/screens/Settings/index.tsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import React from 'react';
-import { View, Text } from 'react-native';
+import { View, Text, StatusBar } from 'react-native';
 
 import styles from './styles';
 
 const Settings = () => (
   <View style={styles.container}>
+    <StatusBar animated backgroundColor="rgba(0, 0, 0, 0.7)" />
```
Comment (guidoprinc) [inline wbooks/src/app/screens/Settings/index.tsx]: extract color into `colors` constant file

File: wbooks/src/app/screens/SearchFilter/index.tsx [hunk 2]
```diff
@@ -29,6 +29,7 @@ const SearchFilter = () => {
 
   return (
     <View style={styles.container}>
+      <StatusBar animated backgroundColor="rgba(0, 0, 0, 0.7)" />
```
Comment (guidoprinc) [inline wbooks/src/app/screens/SearchFilter/index.tsx]: extract color into `colors` constant file

File: wbooks/src/app/screens/Library/index.tsx [hunk 3]
```diff
@@ -21,6 +21,7 @@ const Library = () => {
 
   return (
     <View style={styles.container}>
+      <StatusBar animated backgroundColor="rgba(0, 0, 0, 0.7)" />
```
Comment (guidoprinc) [inline wbooks/src/app/screens/Library/index.tsx]: extract color into `colors` constant file

File: wbooks/src/app/screens/BookDetail/index.tsx [hunk 4]
```diff
@@ -34,17 +34,20 @@ const BookDetail = ({ route }: Props) => {
...
-      ListHeaderComponent={<Information {...route.params} />}
-      data={mock}
-      extraData={mock}
-      renderItem={renderItem}
-      keyExtractor={keyExtractor}
-      ListFooterComponentStyle={styles.footer}
-      ListFooterComponent={Footer}
-    />
+    <>
+      <StatusBar animated backgroundColor="rgba(0, 0, 0, 0.7)" />
```
Comment (guidoprinc) [inline wbooks/src/app/screens/BookDetail/index.tsx]: extract color into `colors` constant file

**Review by guidoprinc — CHANGES_REQUESTED** [review]

File: wbooks/src/app/screens/Settings/index.tsx (hunk 1 above)
Comment (CristianRWolox) [reply wbooks/src/app/screens/Settings/index.tsx]: Done!

File: wbooks/src/app/screens/SearchFilter/index.tsx (hunk 2 above)
Comment (CristianRWolox) [reply wbooks/src/app/screens/SearchFilter/index.tsx]: Done!

File: wbooks/src/app/screens/Library/index.tsx (hunk 3 above)
Comment (CristianRWolox) [reply wbooks/src/app/screens/Library/index.tsx]: Done!

File: wbooks/src/app/screens/BookDetail/index.tsx (hunk 4 above)
Comment (CristianRWolox) [reply wbooks/src/app/screens/BookDetail/index.tsx]: Done!

**Review by guidoprinc — APPROVED** [review]

**Pull request merged**